pkg/controller/backup_controller.go

```diff
@@ -28,7 +28,6 @@ import (
 	"sync"
 	"time"
 
-	"github.com/apex/log"
```

Comment:
Please put this line back. There is still some code using this log library.

Reply:
I am so sorry for the mistake. Please review again.